DTS-STN/Service-Canada-Design-System
# Blur the input element on scroll instead of changing the value in DatePicker component

Merge pull request #511 from DTS-STN/release# DS-##(Jira Issue)-A###(component ID)-(component name)

**Reminder: Add all changes for this PR to the CHANGELOG.md**

Main Reviewer: @zeddotes 

#### Description & Background Context:

Refer to bug "81839: Year of birth textbox content shouldn't be scrollable" of the Eligibility Estimator Sprint board.

#### Where should the reviewer start?

File name: DatePicker.js

#### How should this be tested?

Pull branch and test in different browsers if possible. The behaviour should be: when scrolling wheel of mouse or using trackpad, the dates should not change. Selection should blur away from the input element and the page should scroll up and down when scroll bar is visible. 

#### Are there any breaking changes?

No
